From: Marco Trevisan <mail@3v1n0.net>
Date: Thu, 25 Oct 2018 12:23:15 +0000
Subject: Merge branch 'gdbus-peer-again' into 'master'

gdbus-peer: Make sure to not include objectmanager-gen.c source

See merge request GNOME/glib!416

(cherry picked from commit 4d48e020278b6be64ba84be7b8ab388f5ff2b358)

7c70bef8 gdbus-peer: Make sure to not include objectmanager-gen.c source

Applied-upstream: 2.58.2, commit:4f14e614fac90c9ebedd7a6080089381e18f2d09
---
 gio/tests/gdbus-object-manager-example/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/tests/gdbus-object-manager-example/meson.build b/gio/tests/gdbus-object-manager-example/meson.build
index 6e2cb07..1a87eb4 100644
--- a/gio/tests/gdbus-object-manager-example/meson.build
+++ b/gio/tests/gdbus-object-manager-example/meson.build
@@ -22,5 +22,5 @@ libgdbus_example_objectmanager = library('gdbus-example-objectmanager',
     install_dir : installed_tests_execdir)
 
 libgdbus_example_objectmanager_dep = declare_dependency(
-  sources : gdbus_example_objectmanager_generated,
+  sources : gdbus_example_objectmanager_generated[0],
   link_with : libgdbus_example_objectmanager)
